modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -20,10 +19,27 @@ def index
           render json: claims
         end
 
+        def show
+          claim = ClaimsApi::AutoEstablishedClaim.get_by_id_or_evss_id(params[:id])
+
+          if claim.present?
+            render json: { id: claim[:id], type: claim[:claim_type] }
+          else
+            # If we don't have it, it might still be in BGS, so check there
+            bgs_claim = bgs_service.benefit_claims.find_claim_details_by_claim_id(claim_id: params[:id])
+            claim_details = bgs_claim.dig(:bnft_claim_detail)
+
+            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found') if claim_details.blank?
+
+            render json: { id: claim_details[:bnft_claim_id], type: claim_details[:bnft_claim_type_nm] }
```

Comment:
this logic isnt quite right with what the index endpoint is doing around "type"
should maybe get together with @jw81  and see if there is a way to reuse his mapper class

Reply:
Maybe we could just extract [this logic](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/services/claims_api/bgs_to_lighthouse_claims_mapper_service.rb#L46-#L57) from the mapper service class, into some other thing that can be consumed by the mapper service and anywhere else that we need it.  Would also be good because it'll allow us to have a single place where we build these JSON structures.  Just thoughts.

Reply:
In my [current issue](https://vajira.max.gov/browse/API-8399), I'm switching to consuming a different BGS operation which is going to require me to change this mapping service class a fair amount.  If it's ok with @SlakrHakr, I can go ahead and handle adapting the service class to work for you as well while I'm in there.

Reply:
i think this is the last thing here and im good approving this pr and moving forward if @jw81 is cool doing this work as a part of his ticket 👍🏻

Reply:
cool, adding that as a note on the AC list of that ticket and merging this one